libs/ui-toolkit/src/components/notification/notification.tsx

```diff
@@ -82,7 +82,7 @@ export const Notification = ({
             {title}
           </div>
         )}
-        <div className="text-sm">{message}</div>
+        <div className="text-sm [word-break:break-word]">{message}</div>
```

Comment:
tailwind break-word comes with overflow-wrap and it won't work. https://tailwindcss.com/docs/word-break